### PR TITLE
Fixes Some Night Vision Goggles Issues

### PR DESCRIPTION
**Pull request by ThatDan123**
CL: Fixed night vision goggle not being able to be toggled via action button
CL: Fixed surgery and crayon UI not being draggable

-Cleaned up the night vision script and should now properly sync for late joiners

### CODE REVIEW

File: UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs [hunk 1]
```diff
@@ -870,4 +870,12 @@ public void CmdSetCrayon(GameObject crayon, uint category, uint index, uint colo
 
 		crayonScript.SetTileFromClient(category, index, colourIndex, direction);
 	}
+
+	[Command]
+	public void CmdRequestNightVisionState(GameObject goggles, GameObject player)
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs]: Couldn't this be handled with syncvar?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs]: Changed to sync var

**Comment by Bod9001** [comment]
idk  what  Codacy is complaining about it doesn't say,  say when it's ready to merge, and the code looks good.

**Comment by ThatDan123** [comment]
Is good to merge whenever

**Pull request merged**